infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -792,6 +807,7 @@ public void shutdown() {
         detectionTasksScheduler.shutdownNow();
         detectionTaskWorkers.shutdownNow();
         localMetricsPollingService.shutdownNow();
+        sequencerBootstrapWorker.shutdownNow();
```

Comment:
Not sure, if you are going to keep all four. but in that case, maybe a method that takes a collection of thread pools and shut them might be more readable. (same for awaitTermination)